src/mmds/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2018"
 
 [dependencies]
+chrono = ">=0.4.19"
```

Comment:
I know we removed `chrono` a while back since `std` provided everything we need. Can we do this without it?